docs/source/data-types/timeuuid.md

```diff
@@ -19,6 +19,45 @@ session
     .query("INSERT INTO keyspace.table (a) VALUES(?)", (to_insert,))
     .await?;
 
+// Read timeuuid from the table
+if let Some(rows) = session.query("SELECT a FROM keyspace.table", &[]).await?.rows {
+    for row in rows.into_typed::<(CqlTimeuuid,)>() {
+        let (timeuuid_value,): (CqlTimeuuid,) = row?;
+    }
+}
+# Ok(())
+# }
+```
+
+To create your own `Timeuuid` objects from timstamp-base `uuid` v1, you need to enable the feature `v1` of `uuid` crate using:
+
+```shell
+cargo add uuid -F v1
+```
+
+and now you're gonna be able to use the `uuid::v1` features: 
+
+```rust
+# extern crate uuid;
+# extern crate scylla;
+# use scylla::Session;
+# use std::error::Error;
+# use std::str::FromStr;
+use scylla::IntoTypedRows;
+use scylla::frame::value::CqlTimeuuid;
+use uuid::Uuid;
+# async fn check_only_compiles(session: &Session) -> Result<(), Box<dyn Error>> {
+
+// Tip: you can use random stable numbers or your MAC Address
+let node_id = [0x12, 0x34, 0x56, 0x78, 0x9A, 0xBC];
```

Comment:
Perhaps leaving here a link to Wikipedia on UUID v1 is a good idea:
https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_1_(date-time_and_MAC_address)

Reply:
After some chatting with @codelieutenant he gave the idea to also point as an environment variable. Since it's a thing that it's not supposed to change maybe it's suppose to be a good fit.

What do you think about this approach?

Reply:
Seems legit, though again - I have no idea what people normally do to acquire the node id in their applications.